Makefile

```diff
@@ -19,7 +18,7 @@ install: build
 	golangci-lint --version
 
 lint:
-	golangci-lint run -v -E $(ENABLED_LINTERS) --fix $(PKG)
+	golangci-lint run --fix
```

Comment:
I would prefer to keep the -v verbose mode, to know exactly which linters are used.